modules/stochastic_tools/test/tests/multiapps/sampler_multiapp/tests

```diff
@@ -1,7 +1,18 @@
 [Tests]
-  [./run]
+  [./transient]
     type = CheckFiles
-    input = master.i
-    check_files = 'master_out_runner0.e master_out_runner1.e master_out_runner2.e master_out_runner3.e master_out_runner4.e'
+    input = master_transient.i
+    check_files = 'master_transient_out_runner0.e master_transient_out_runner1.e master_transient_out_runner2.e master_transient_out_runner3.e master_transient_out_runner4.e'
+    issues = '9923'
+    design = 'SamplerTransientMultiApp.md'
+    requirement = 'MOOSE shall provide an ability to creates a transient type sub application from the Sampler object.'
+  [../]
+  [./full_solve]
+    type = CheckFiles
+    input = master_full_solve.i
+    check_files = 'master_full_solve_out_runner0.e master_full_solve_out_runner1.e master_full_solve_out_runner2.e master_full_solve_out_runner3.e master_full_solve_out_runner4.e'
+    issues = '9923'
```

Comment:
Should have a '#' in front of these ticket numbers. Right @aeslaughter?

Reply:
```suggestion
    issues = '#9923'
```